_includes/footer.html

```diff
@@ -16,12 +16,12 @@ <h4>About IEEE</h4>
                 <h4 class="secondary">Information</h4>
                 <p>
                     <ul class="colorlib-footer-links">
-                        <li><a href="{{ site.url }}"><i class="fa fa-check"></i> Home</a></li>
+                        <li><a href="index.html"><i class="fa fa-check"></i> Home</a></li>
```

Comment:
We don't wanna hard-code links here. We should be able to dynamically render them from `header.json`. If you changed this because the existing implementation didn't work, please try to fix the issue and let me know 👍🏻.

Reply:
![image](https://user-images.githubusercontent.com/78093264/124770782-3ee14980-df58-11eb-9cc9-eb70442cc7ce.png)
changed to " \ "